pyfr/partitioners/base.py

```diff
@@ -340,21 +299,21 @@ def partition(self, mesh):
         # Generate a new UUID for the mesh
         newmesh['mesh_uuid'] = newuuid = str(uuid.uuid4())
 
-        # Annotate the shape point arrays
-        self._prepare_attrs(newmesh)
-
         # Build the solution converter
         def partition_soln(soln):
             # Check the UUID
             if curruuid != soln['mesh_uuid']:
                 raise ValueError('Mismatched solution/mesh')
 
+            # Obtain the prefix
+            prefix = Inifile(soln['stats']).get('data', 'prefix')
```

Comment:
If I understood correctly, this should to allow repartition files with temporal average data isn't it? That's I nice addition

Reply:
Yes.